src/actions/riderActions.js

```diff
@@ -1,17 +1,19 @@
 import * as types from "./actionTypes";
 import axios from "axios";
+import * as api from "../util/api.js";
+import store from "../store";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'store'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)